graphjet-core/src/main/java/com/twitter/graphjet/algorithms/counting/TopSecondDegreeByCountRequest.java

```diff
@@ -77,6 +79,33 @@ public TopSecondDegreeByCountRequest(
     this.maxTweetSocialProofSize = maxTweetSocialProofSize;
     this.minUserSocialProofSizes = minUserSocialProofSizes;
     this.resultFilterChain = resultFilterChain;
+    this.socialProofTypeUnions = Collections.emptySet();
+  }
+
+  public TopSecondDegreeByCountRequest(
```

Comment:
I was only worried that this introduces a backwards-incompatible change and that everyone who uses this request needs to update their code when this goes out. What do you think?